src/Bonsai.ML.Torch/Ones.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.ComponentModel;
+using System.Reactive.Linq;
+using static TorchSharp.torch;
+
+namespace Bonsai.ML.Torch
+{
+    /// <summary>
+    /// Creates a tensor filled with ones.
+    /// </summary>
+    [Combinator]
+    [Description("Creates a tensor filled with ones.")]
+    [WorkflowElementCategory(ElementCategory.Source)]
+    public class Ones
+    {
+        /// <summary>
+        /// The size of the tensor.
+        /// </summary>
+        [Description("The size of the tensor.")]
+        public long[] Size { get; set; } = [0];
```

Comment:
For these types of properties you might want to look into the `UnidimensionalArrayConverter` converter class, as it gives you a simple way to write arrays as comma-separated values in the property grid.

By the way, respectively you have `MultidimensionalArrayConverter` for two-dimensional arrays (which now makes me think probably would be better to have called it `RectangularArrayConverter`).